### DISCUSSION TITLE
这么去掉这个？

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#31] [Q&A]
![Sin título](https://github.com/densen2014/docs/assets/122285019/a855f4a2-ef46-4171-be06-1dd85609a11a)

**Comment by densen2014** [comment]
1. 如果是昨天没问题, 那就恢复到昨天的配置文件
2. 设置=>主界面=>商品显示库位管理关掉

**Reply by aaronameixa** [reply]
![image](https://github.com/densen2014/docs/assets/122285019/f83db6c1-3526-4748-af85-0c08d5b8e8cc)
找不到(主界面)

**Reply by densen2014** [reply]
就是设置里面第一页

**Comment by densen2014** [comment]
@aaronameixa 如果解决了及时关闭

![image](https://github.com/densen2014/docs/assets/8428709/3a90336b-4f65-4628-8cbb-06d5f501d83b)
